README.md

```diff
@@ -6,17 +6,32 @@ Release automation for pandas.
   - [  ] `TAG` in `Makefile`
   - [  ] `GH_USERNAME` in `Makefile`
 
-If running for the first time be sure to initialize repos
+## **If running for the first time:**
 
+- Create a conda enviroment based on environment.yml:
+```sh
+conda env create 
```

Comment:
Shouldn't this be `conda env create -f environment.yaml`?

Reply:
`environment.yml` is the default conda file name, so not needed.

Reply:
Personally, even if it is the default, I find leaving it out just confusing (for all people that don't know or directly remember that this is the default)

Reply:
ok, I see the non-explicit form is confusing to people. will change it back :D